jolly-jellyfish/src/django_meta/urls.py

```diff
@@ -0,0 +1,13 @@
+"""halfway URL Configuration"""
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.contrib.auth import views as auth_views
+from django.urls import path, include
+
+urlpatterns = [
+    path('admin', admin.site.urls),
+    path('users/login', auth_views.LoginView.as_view(), name='login'),
+    path('users/logout', auth_views.LogoutView.as_view(next_page='/'), name='logout'),
+    path('', include('page_maker.urls')),
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Can you justify this? Why wouldn't this just be inside the list instead of this weird concatenation thing? Seems strange to me.

Reply:
I didn't write this bit, but it seems to be recommended by Django - https://docs.djangoproject.com/en/3.1/howto/static-files/#serving-static-files-during-development.

Reply:
I would assume that `static` returns a list rather something to be an element of that list, so to include it in the first list would require unpacking?

Reply:
ah, right you are. I recommend looking into the Whitenoise project for handling static file serving in Django, we use it for all our projects.